pages/Messages/Ordering.md

```diff
@@ -0,0 +1,29 @@
+---
+name: Ordering
+route: /Messages/Ordering
+menu: Messages
+---
+
+# Messages Ordering
+
+## Specification Status
+
+| Version | Status |
+---------- | ---------
+| 0.1     | Tentative |
+
+## Purpose
+
+1. Describe a standard for ordering DSNP messages and resolving conflicts in DSNP message with time dependent context.
+
+## Assumptions
+
+* All assumptions from [DSNP Messages](/Messages/Overview)
+
+## Ordering
+
+DSNP messages must be ordered at the highest level by the block number of the block in which a message's batch is announced with higher block numbers indicating more recent messages and lower block numbers representing older ones.
+After block number, announcements published in the same block must be ordered by the transaction index of the block with higher transaction indices indicating more recent messages and lower transaction indices indicating older ones.
```

Comment:
Should this be "transaction index of the transaction", "transaction index of the announcement", or just "transaction index"?  "transaction index of the block" doesn't make any sense to me.

Reply:
Fixed